ansible/inventory/group_vars/apps.yaml

```diff
@@ -212,20 +213,18 @@ dns_records:
     ttl: 300
 
 stack_manager:
-  ssm_stack_name: ssm
+  ssm_stack_name: aem-stack-manager-ssm
   stack_name: aem-stack-manager
   s3_prefix: stack-manager
-  utilities_stack_name: utilities
-  purge:
-    live_snapshots:
-      schedule: '10 20 1/3 * ? *'
-      max_age_in_hours: 24
-    offline_snapshots:
-      schedule: '15 19 ? * SUN *'
-      max_age_in_hours: 61320
-    orchestration_snapshots:
-      schedule: '5 0/4 * * ? *'
-      max_age_in_hours: 4
+
+snapshots_purge:
```

Comment:
This is an incorrect merge. The new structure for snapshot purging under stack_manager property is the correct one as per configuration doco.